general-superstaq/general_superstaq/superstaq_client_test.py

```diff
@@ -287,20 +291,25 @@ def test_superstaq_client_create_job_json(mock_post: mock.MagicMock) -> None:
         )
 
 
-@mock.patch("requests.get")
-def test_superstaq_client_get_job(mock_get: mock.MagicMock) -> None:
-    mock_get.return_value.ok = True
-    mock_get.return_value.json.return_value = {"foo": "bar"}
+@mock.patch("requests.post")
+def test_superstaq_client_fetch_jobs(mock_post: mock.MagicMock) -> None:
+    mock_post.return_value.ok = True
+    mock_post.return_value.json.return_value = [{"foo": "bar"}]
     client = gss.superstaq_client._SuperstaqClient(
         client_name="general-superstaq",
         remote_host="http://example.com",
         api_key="to_my_heart",
     )
-    response = client.get_job(job_id="job_id")
-    assert response == {"foo": "bar"}
-
-    mock_get.assert_called_with(
-        f"http://example.com/{API_VERSION}/job/job_id", headers=EXPECTED_HEADERS, verify=False
+    response = client.fetch_jobs(job_ids=["job_id"], cq_token="token")
```

Comment:
shouldn't this be a dictionary like
```suggestion
    response = client.fetch_jobs(job_ids=["job_id"], cq_token={"access_token": "token"})
```